jQuery/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  margin: 20px 5%;
+}
+
+#hh-menu {
+  border-right: solid 1px black;
+  position: fixed;
+  width: 240px;
+  height: 100%;
+}
+
+#hh-menu > select {
```

Comment:
The margin stuff felt wrong and I specifically hoped you would address it. I'm interested to learn the better way of doing it